[2023-10-11 15:34:35,577] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: tf_pipeline.data_validation 2023-10-11T15:33:42.973269+00:00 [queued]>
[2023-10-11 15:34:35,620] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: tf_pipeline.data_validation 2023-10-11T15:33:42.973269+00:00 [queued]>
[2023-10-11 15:34:35,621] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-10-11 15:34:35,621] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2023-10-11 15:34:35,621] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-10-11 15:34:35,645] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): data_validation> on 2023-10-11T15:33:42.973269+00:00
[2023-10-11 15:34:35,672] {standard_task_runner.py:52} INFO - Started process 7929 to run task
[2023-10-11 15:34:35,680] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'tf_pipeline', 'data_validation', '2023-10-11T15:33:42.973269+00:00', '--job-id', '7', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpbengd5no', '--error-file', '/tmp/tmp1k0yc7lr']
[2023-10-11 15:34:35,691] {standard_task_runner.py:77} INFO - Job 7: Subtask data_validation
[2023-10-11 15:34:35,844] {logging_mixin.py:104} INFO - Running <TaskInstance: tf_pipeline.data_validation 2023-10-11T15:33:42.973269+00:00 [running]> on host 0e092fd43c1f
[2023-10-11 15:34:35,954] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tf_pipeline
AIRFLOW_CTX_TASK_ID=data_validation
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T15:33:42.973269+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-11T15:33:42.973269+00:00
[2023-10-11 15:34:35,961] {python.py:151} INFO - Done. Returned value was: None
[2023-10-11 15:34:35,984] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=tf_pipeline, task_id=data_validation, execution_date=20231011T153342, start_date=20231011T153435, end_date=20231011T153435
[2023-10-11 15:34:36,044] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-11 15:34:36,099] {local_task_job.py:151} INFO - Task exited with return code 0
